Update the licensing pages (#8345)
Particular/docs.particular.net: servicecontrol/license.md

---
title: Licensing ServiceControl
summary: Configure and manage ServiceControl license
reviewed: 2026-06-16
component: ServiceControl
related:
- servicepulse/license
- nservicebus/licensing
---
## Behavior

The license is read only once at startup.

### Expired license

> If the license has expired (or is corrupted), existing ServiceControl instances will continue working undisrupted and could be started and stopped normally.

Reviewer: ```suggestion
If the license has expired (or is corrupted), existing ServiceControl instances will continue to function and can be started and stopped normally.
```

> However, no new instances could be added.

Reviewer: Is this true? What prevents a new installation of ServiceControl? If SCMU prevents it, do the PowerShell modules also prevent this?

If this is true, then here's a small nitpick suggestion:
```suggestion

New instances cannot be added.
```

---
title: Licensing ServiceControl
summary: Configure and manage ServiceControl license
reviewed: 2026-06-16
component: ServiceControl
related:
- servicepulse/license
- nservicebus/licensing
---
## Behavior

> The license is read only once at startup.

Reviewer 2: In dev it's read for every license request from ServicePulse in addition to startup, not sure if this is different for Prod
